includes/admin/helper/class-wc-helper-updater.php

```diff
@@ -162,6 +165,93 @@ public static function get_update_data() {
 		return self::_update_check( $payload );
 	}
 
+	/**
+	 * Get translations updates informations.
+	 *
+	 * Scans through all subscriptions for the connected user, as well
+	 * as all Woo extensions without a subscription, and obtains update
+	 * data for each product.
+	 *
+	 * @return array Update data {product_id => data}
+	 */
+	public static function get_translations_update_data() {
+		$payload = array();
+
+		$installed_translations = wp_get_installed_translations( 'plugins' );
+
+		$locales = array_values( get_available_languages() );
+		/**
+		 * Filters the locales requested for plugin translations.
+		 *
+		 * @since 3.7.0
+		 * @since 4.5.0 The default value of the `$locales` parameter changed to include all locales.
+		 *
+		 * @param array $locales Plugin locales. Default is all available locales of the site.
+		 */
+		$locales = apply_filters( 'plugins_update_check_locales', $locales );
+		$locales = array_unique( $locales );
+
+		// Scan local plugins which may or may not have a subscription.
+		$plugins = WC_Helper::get_local_woo_plugins();
+		$active      = array_intersect( array_keys( $woo_plugins ), get_option( 'active_plugins', array() ) );
+
+		$to_send = compact( 'plugins', 'active' );
+
+		if ( wp_doing_cron() ) {
+			$timeout = 30;
+		} else {
+			// Three seconds, plus one extra second for every 10 plugins.
+			$timeout = 3 + (int) ( count( $plugins ) / 10 );
+		}
+
+		$options = array(
+			'timeout'    => $timeout,
+			'body'       => array(
+				'plugins'      => wp_json_encode( $to_send ),
+				'translations' => wp_json_encode( $translations ),
+				'locale'       => wp_json_encode( $locales ),
+				'all'          => wp_json_encode( true ),
+			),
+			'user-agent' => 'WordPress/' . $wp_version . '; ' . home_url( '/' ),
+		);
+
+		if ( $extra_stats ) {
+			$options['body']['update_stats'] = wp_json_encode( $extra_stats );
+		}
+
+		$url      = 'htts://translate.wordpress.com/projects/';
```

Comment:
`htts` is not a valid protocol ;)

Reply:
It is a dummy placeholder. No worries. Good spotting!